apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/FissClaimRdaSink.java

```diff
@@ -114,9 +114,15 @@ RdaClaimMessageMetaData createMetaData(RdaChange<RdaFissClaim> change) {
   MessageError createMessageError(
       String apiVersion, FissClaimChange change, List<DataTransformer.ErrorMessage> errors)
       throws IOException {
-    return MessageError.builder()
+    MessageError.MessageErrorBuilder builder = MessageError.builder();
+
+    // Base64 encode the claim key so it matches any existing claim records in the database
+    DataTransformer dt = new DataTransformer();
+    dt.copyBase64String(
+        "rdaClaimKey", false, 1, 43, 23, change.getClaim().getRdaClaimKey(), builder::claimId);
+
```

Comment:
23 is the DCN length.  The overall field would be at least 28 (DCN+intermediaryNb)